mish_mash_utils.ts

```diff
@@ -52,10 +53,10 @@ export const renderMishMashChoice = () => {
 const findCorrectRecipe = () => {
     selectedProducts = $('.recipes__products--select').toArray().map(object => object.innerHTML)
     recipes.forEach(recipes => {
-        const correctProducts = recipes.recipe.products.every(product => selectedProducts.includes(product))
+        const correctProducts = recipes.products.every((product:string) => selectedProducts.includes(product))
```

Comment:
you don't need to add type here, as soon as you `Recipe` is correctly typed TypeScript will know that product is a string